### ISSUE TITLE
Bug: it is very slow to open the terminal after installing sdkman

### ISSUE THREAD

**Issue by jiangtao69039** [sdkman/sdkman-cli#1050]
**Bug report**
even if i set offline mode disable, it is also open a terminal very slow.

its seem like exec command  is blocked by some network access. 

if i disable wifi and networking, the sdk command will exec so fast.

**To reproduce**
Used in China, some websites will be restricted

**System info**
- Arch
- zsh
- 5.13.1
-->

**Comment by jiangtao69039** [comment]
![sdk](https://user-images.githubusercontent.com/20572131/154475169-7d7d2710-ccd7-45de-925e-c2e9ae25587b.png)

**Comment by jcrben** [comment]
@jiangtao69039 how was this resolved? it's fast for you now? In 5.16 in "bash for windows" it's pretty slow still

**Comment by jiangtao69039** [comment]
> @jiangtao69039 how was this resolved? it's fast for you now? In 5.16 in "bash for windows" it's pretty slow still

When I finish installing the software, I turn on offline mode. It may be that the network access of sdkman is time-consuming.

I upgrade to the latest version and append command in ~/.bashrc :
```
#THIS MUST BE AT THE END OF THE FILE FOR SDKMAN TO WORK!!!
export SDKMAN_DIR="$HOME/.sdkman"
[[ -s "$HOME/.sdkman/bin/sdkman-init.sh" ]] && source "$HOME/.sdkman/bin/sdkman-init.sh"
sdk offline enable >> /dev/null
```

**Comment by jverne** [comment]
For me, it is sourcing sdkman-init,sh that takes up to 10s when I launch a login shell. Until this is done "sdk" can't be run to enable offline mode.
